bundles/org.openhab.binding.homematic/src/main/resources/OH-INF/i18n/homematic_de.properties

```diff
@@ -49,6 +49,12 @@ thing-type.config.homematic.bridge.wiredPort.label = Wired Port
 thing-type.config.homematic.bridge.wiredPort.description = Die Portnummer des HS485-Dienstes
 thing-type.config.homematic.bridge.xmlCallbackPort.label = XML-RPC Callback Port
 thing-type.config.homematic.bridge.xmlCallbackPort.description = Callback-Port des XML-RPC-Servers. Wenn kein Wert angegeben ist, startet xmlCallbackPort mit 9125 und zählt hoch
+thing-type.config.homematic.bridge.password.label = Password für Zugriff auf das Gateway
```

Comment:
Please remove this file as translations should be added by the use of crowdin.

Reply:
Done